docs/FAQ/gui-faq-en.md

```diff
@@ -4,163 +4,178 @@ Zonemaster
-who has run a specific test since nothing is logged except the time of the test.
+domain and its history of tests.
+However there is no way to tell who has run a specific test since nothing more than the test
+parameters and results are stored.
+Specifically, no cookies or information on the user's IP address is stored in the database.
+The user who initiated the test cannot be traced back from the information in the database.
 
 <a name="q10"></a>
-#### 10. How come I cannot test my domain?
-If we disregard the situation where the domain does not exist, as in when you input a
-non-existing domain to Zonemaster, there are 2 other possibilites:
-  - To protect the engine from multiple identical inputs, that is the same IP
-    checking the same zone several times, there is a delay of 5 minutes between
-    identical subsequent tests. This means that you can only test the
-    same domain once every 5 minutes, if you try and test it again within 5 minutes
-    the last results will be displayed instead.
-  - Because Zonemaster was made to check domains or DNS zones, like zonemaster.net, and
-    not hostnames in a domain (zone), like www.zonemaster.net, the Zonemaster will
-    report a failure if you try to test a host name instead of a domain matching a
-    DNS zone.
+#### 10. How come my domain name cannot be tested?
+There are several possibilities:
+
+- Your domain name is not yet delegated.
+- Your domain name is not reachable from public Internet.
+- Zonemaster can only test what is called a DNS zone (e.g. 'zonemaster.net') and not host names (e.g. 'www.zonemaster.net')
+- There is a 10 minutes protection between consecutive tests for a given domain name (with same test parameters).
+  Running a test within that window will instead show the last available test for that domain name (and parameters).
+- You have misspelled your domain name.
 
 <a name="q11"></a>
 #### 11. What kind of queries does Zonemaster generate?
-Zonemaster send multiple DNS queries to the name servers hosting the domain name and
-also to the name servers hosting the parent zone of the domain name.
+Zonemaster send multiple DNS queries to the name servers hosting the domain name being tested and
+also to the name servers hosting the parent zone of that domain name.
 
-To get a full
-view of what queries and results are generated you can run the
-CLI interface (and in order to run the CLI interface you need to download the
-complete package and install it) and select full output.
-The output from the CLI tool is quite heavily technical
-so unless you are into bits and bytes you might want to skip this step.
+The GUI interface of Zonemaster does not show any queries sent.
+Only the CLI interface can. If you want to see such queries, you will have to locally install
+a minimally working Zonemaster instance with both the Engine and CLI components.
+Queries sent can be shown using the 'DEBUG' level option.
+Fair warning, the output from the CLI can be quite heavy.
```

Comment:
I suggest adding reference to Docker and the "Using the CLI" page:
```
Only the CLI interface can. If you want to see such queries, you will have to locally install
a minimally working Zonemaster instance with both the Engine and CLI components or install
Docker and use the Zonemaster Docker image.
Queries sent can be shown using the 'DEBUG' level option.
Fair warning, the output from the CLI can be quite heavy. For more information see
[Using The CLI].
```

Reply:
Updated (rephrased a bit)